plugin/core/sessions.py

```diff
@@ -1513,6 +1513,8 @@ def _open_uri_with_plugin_async(
             result = Promise.packaged_task()  # type: PackagedTask[Optional[sublime.View]]
 
             def open_scratch_buffer(title: str, content: str, syntax: str) -> None:
+                if group > -1:
+                    self.window.focus_group(group)
```

Comment:
I'll admit that I'm not that familiar with group handling in ST. Can you explain why this is needed? I thought that `new_file` takes care of focusing new group (unless maybe `FORCE_GROUP` is specified... not clear about how that works.).

Reply:
Never mind. `new_file` doesn't accept `group`, only `open_file`.